civis/_deprecation.py

```diff
@@ -1,7 +1,7 @@
 from functools import wraps
 import warnings
 
-from civis.compat import signature
+from .compat import signature
```

Comment:
Why change this line? If the tests pass, then it should be okay, but there's some differences in how relative imports are handled between Python 2 and 3, and explicit imports tend to be more robust.

Reply:
The previous way led to a test failure because of adding the import for `deprecate_param` to `civis.py`.  I believe the issue is that this module is imported by `civis.py`, but then the `from civis.compat` version causes `civis.py` to be imported again.  The relative import doesn't lead to a cycle for some reason.


From [travis](https://travis-ci.org/civisanalytics/civis-python/builds/353073967), before I changed the line in question:

```
Traceback:
civis/__init__.py:4: in <module>
    from civis.civis import APIClient, find, find_one
civis/civis.py:7: in <module>
    from civis.resources import generate_classes_maybe_cached
civis/resources/__init__.py:1: in <module>
    from ._resources import (generate_classes,
civis/resources/_resources.py:17: in <module>
    from civis._deprecation import deprecate_param
civis/_deprecation.py:4: in <module>
    from civis.compat import signature
E   ImportError: No module named compat
```

Reply:
@stephen-hoover, does that answer your question?

Reply:
Yes, thank you!